src/Abp.EntityFramework/EntityFramework/Repositories/EfRepositoryBaseOfTEntityAndTPrimaryKey.cs

```diff
@@ -96,37 +95,37 @@ public override IQueryable<TEntity> GetAllIncluding(params Expression<Func<TEnti
             return query;
         }
 
-        public override async Task<List<TEntity>> GetAllListAsync()
+        public override async Task<List<TEntity>> GetAllListAsync(CancellationToken cancellationToken = default)
```

Comment:
This should be `CancellationToken?` without `= default` if we want to use `CancellationTokenProvider.Token` in `GetCancellationToken(cancellationToken)` by default.

Reply:
@acjh isn't it the case for other methods ? For example this one https://github.com/aspnetboilerplate/aspnetboilerplate/pull/5111/files/afae99a113590dc8bf5330487d080ca513a43e01#diff-7dcdc5ef60a375982274b9917a12d971R108 ? If we don't use `CancellationToken?`, `CancellationTokenProvider.None` will be used.

Reply:
Yes. I mean change for all methods. Otherwise `cancellationToken` will never be `null` and `CancellationTokenProvider.Token` will never be used.